ee/frontend/mobile-replay/mobile.types.ts

```diff
@@ -122,6 +122,57 @@ type wireframeBase = {
     style?: MobileStyles
 }
 
+export type wireframeInputBase = wireframeBase & {
+    type: 'input'
+    disabled: boolean
+}
+
+export type wireframeCheckBox = wireframeInputBase & {
+    inputType: 'checkbox'
+    checked: boolean
+}
+
+export type wireframeRadio = wireframeInputBase & {
+    inputType: 'radio'
+    checked: string
```

Comment:
```suggestion
    checked: boolean
```

Reply:
In HTML this will expand to multiple radio inputs

So, we'd have as an example

```
{
 <... snip base/>
   inputType: 'radio,
   checked: 'huey',
   values: ['huey', 'duey', 'lewey'],
}
```

to html

```
  <div>
    <input type="radio" id="huey" name="some-uuid" value="huey" checked />
    <label for="huey">Huey</label>
  </div>

  <div>
    <input type="radio" id="dewey" name="some-uuid" value="dewey" />
    <label for="dewey">Dewey</label>
  </div>

  <div>
    <input type="radio" id="louie" name="some-uuid" value="louie" />
    <label for="louie">Louie</label>
  </div>`
```
Makes me realise we need to support value and label

Reply:
Yep, if that's easier for the mobile 👍 

So long as the children don't need to know about each other and the parent can pass knowledge down I'm easy :)